src/Workspaces/CSharp/Portable/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -180,7 +180,7 @@ static SyntaxNodeExtensions()
             s_bannerMatcher =
                 Matcher.Sequence(
                     Matcher.OneOrMore(commentLine),
-                    s_oneOrMoreBlankLines);
+                    Matcher.Repeat(singleBlankLine));
```

Comment:
Should `s_oneOrMoreBlankLines` go away? I see it's not unused after this change, but a few of the uses mention "banner" in the word and so I'm not sure if more changes are needed.